src/routes/GroupSettings/ExportDataTab/ExportData.store.js

```diff
@@ -0,0 +1,12 @@
+import { getHost } from 'store/middleware/apiMiddleware'
+
+export const MODULE_NAME = 'ExportData'
+export const EXPORT_MEMBERS = `${MODULE_NAME}/EXPORT_MEMBERS`
+
+export function requestMemberCSV (groupId) {
+  window.location.assign(`${getHost()}/noo/export/group?groupId=${groupId}&datasets[]=members`)
```

Comment:
this url doesnt make total sense to me. if you are going to have path of /group then seems like the groupId should be a path parameter too? or leave out the /group part and just check for parameter groupId to determine that this is a single group export? not a big deal just curious about the choice